modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -16,25 +16,6 @@
 
 <%@ include file="/init.jsp" %>
 
-<liferay-ui:tabs
-	names="Alerts,Badges,Buttons,Cards,Dropdowns,Form Elements,Icons,Labels,Links,Management Toolbars,Navigation Bars,Pagination Bars,Progress Bars,Stickers"
-	refresh="<%= false %>"
->
+<div id="perico"></div>
 
-	<%
-	String[] sections = {"alerts", "badges", "buttons", "cards", "dropdowns", "form_elements", "icons", "labels", "links", "management_toolbars", "navigation_bars", "pagination_bars", "progress_bars", "stickers"};
-
-	for (int i = 0; i < sections.length; i++) {
-	%>
-
-		<liferay-ui:section>
-			<clay:container-fluid>
-				<liferay-util:include page='<%= "/partials/" + sections[i] + ".jsp" %>' servletContext="<%= application %>" />
-			</clay:container-fluid>
-		</liferay-ui:section>
-
-	<%
-	}
-	%>
-
-</liferay-ui:tabs>
\ No newline at end of file
+<script src="/o/frontend-taglib-clay-sample-web/index.js" type="module" />
```

Comment:
This is how you bootstrap a browser module. 

It is the equivalent of `<aui:script require="<%= npmResolvedPackageName + ...%>">`

In the future, we could make a new tag to avoid using `<aui:script>` to be able to make our own magic before printing the `<script>` tag to the HTML, but it would be straightforward, most probably.

Reply:
> to make our own magic before printing

By magic you mean pointing to a valid resolved path(with version of the module and a path to /o/) like the generated NPMResolver ones?

Reply:
By magic I meant "anything we want to do with it, now or in the future". So I'm proposing to have our own taglib just for the sake of having an extra layer between the JSP and the actual `<script>` tag.

I don't know if in this browser modules solution we would need to have version numbers or not :thinking: In this case, the canonical names of modules contains the `Web-ContextPath` of the OSGi bundle containing them, and that has to be unique per OSGi bundle version, so maybe we can switch from npm versions to OSGi locations to designate which npm modules we want to use. It's something that we should investigate.

Most probably, we would want to do things like:

- `<new:script-tag import="/index.js"/>` to import local (contained in the OSGi bundle contanining the JSP's web context) modules.
- `<new:script-tag import="react" context="frontend-js-react-web" />` to import a module contained in a different web context (if we wanted to allow for it).

Or, altenatively, we could have some way to configure mappings from logical package names to web context + name which are then applied when building. So, doing something like: 

``<new:script-tag import="react" />` in your code, then having a file where it says that `react` has to be loaded from `frontend-js-react-web`

Nothing is yet decided, but those are the possibilities I have currently in mind...

Reply:
This is very interesting @izaera, I also thing that having this tag would probably make things easier. Can't wait to see the final result

Reply:
> Can't wait to see the final result

Well, you will have too, because this is only a brainstorming, nothing that we have decided or planned yet :sweat_smile: